package.json

```
@@ -53,38 +56,49 @@
     "./javascript.{js,d.ts,js.map}",
     "./lit.{js,d.ts,js.map}",
     "./prettier.{js,d.ts,js.map}",
+    "./react.{js,d.ts,js.map}",
+    "./react-perf.{js,d.ts,js.map}",
     "./rules",
     "./sort.{js,d.ts,js.map}",
     "./testing.{js,d.ts,js.map}",
+    "./tsconfig.recommendation.json",
```

The file name seems to be wrong.
```suggestion
    "./tsconfig.recommended.json",
```